app/src/main/kotlin/io/eugenethedev/taigamobile/data/repositories/TasksRepository.kt

```diff
@@ -27,10 +27,11 @@ class TasksRepository @Inject constructor(
         type = statusType
     )
 
-    override suspend fun getFiltersData(commonTaskType: CommonTaskType) = withIO {
+    override suspend fun getFiltersData(commonTaskType: CommonTaskType, isCommonTaskFromBacklog: Boolean) = withIO {
```

Comment:
That's not correct at all. You need to provide string value `"null"` here in case if this is a backlog (which you've implemented right), but in case if this is a sprint (milestone) you need to provide its id to get correct filters data. So instead of `isCommonTaskFromBacklog: Boolean` argument you need to introduce something like `sprintId: Long? = null` here.